pkg/ipamd/rpc_handler.go

```diff
@@ -179,7 +179,7 @@ func (s *server) AddNetwork(ctx context.Context, in *rpc.AddNetworkRequest) (*rp
 	}
 
 	if s.ipamContext.enablePodIPAnnotation {
-		s.ipamContext.AnnotatePod(in.K8S_POD_NAME, in.K8S_POD_NAMESPACE, vpccniPodIPKey, ipv4Addr)
```

Comment:
nit: let's add a comment here that on ADD, we pass empty string as there is no IP being released